daemon/main.c

```diff
@@ -11,8 +15,40 @@
 #include <syslog.h>
 #include <unistd.h>
 
+int get_unused_task(int used_tasks[MAX_TASKS])
```

Comment:
deci aici , daca inteleg bine, daca nu ai thread available iti da eroare si iti zice ca nu s-a putut adauga taskul nu?

daca ce am zis mai sus e adevarat, propun sa punem pe extra-to-do-list urmatoarea chestie:

cand nu ai thread available sa adaugi taskul intr-un queue din care iei atunci cand termini alt task si iti apare un thread available